x/rollup/keeper/deposits.go

```diff
@@ -215,25 +228,33 @@ func (k *Keeper) mintETH(
 		return nil, fmt.Errorf("transfer amount %v is greater than mint amount %v", transferAmount, mintAmount)
 	}
 
+	recipientAddrSDK, err := sdk.AccAddressFromBech32(recipientAddr)
```

Comment:
nit: could you rename `recipientAddrSDK` to `recipientSDKAddr` to have parity with `mintSDKAddr`?